etc/deposit.properties

```diff
@@ -1,6 +1,8 @@
 # Fedora Commons Repository (https, localhost, 443, fedora, fedoraAdmin, ??)
 project.version=4.0
 
+fcrepo.baseUrl=http://localhost:8080/fcrepo/rest
```

Comment:
do we really need both of these properties? i'm concerned that !) they are nearly identical and therefore easily confused and 2) that they are both really URLs